### PR TITLE
Indicate when local layers were used and from where

### LINKED ISSUE #258: charm-build -l DEBUG should tell me where it found the layer its using
## Checklist

 - [X] Confirmed this is an issue with charm-tools, not charmstore-client
 - [ ] Provide versions of tools used
 - [ ] Described the feature or ways to replicate the issue

## What version am I running?

I ran the following command: `charm version` and got the following output:

```
charm error
charm-tools 2.1.2

```

I am using:  Ubuntu 16.04 (snap install charm)

## Issue/Feature

When I charm build -l DEBUG i would expect to see some form of output indicating where the layer came from. Either full path on disk that it located or perhaps the github repository for remote layers coming from the interface API server.

This however doesn't appear to be the case, and debugging incorrect layer inclusions is difficult.

### PR DESCRIPTION
**Pull request by johnsca**
Description of change

## Checklist

 - [x] Have you followed [Juju Solutions hacking guide?](https://hacking.juju.solutions)
 - [x] Are all your commits [logically] grouped and squashed appropriately?
 - [ ] Does this patch have code coverage?
 - [x] Does your code pass `make test`?

Fixes #258

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -21,8 +21,9 @@ commands = coverage erase
            coverage run --include=charmtools/* {envbindir}/nosetests {posargs}
            coverage report
 deps =
-    {toxinidir}
+    -e{toxinidir}
     -r{toxinidir}/requirements.txt
+    ipdb
```
Comment (johnsca) [inline tox.ini]: I didn't mean to commit these, but they make debugging tests nicer, so I'm leaving it in as a drive-by.

**Comment by chuckbutler** [comment]
Despite the missing code coverage, I am +1 to this change as it clearly indicates where an assembled layer is coming in from. 👍

**Comment by mbruzek** [comment]
+1 from me. We had build time issues that this would have fixed.
